server/src/testE2E/java/com/flipkart/varadhi/E2EBase.java

```diff
@@ -263,27 +265,31 @@ static <T> Response makeHttpPostRequest(String targetUrl, T entityToCreate) {
         return getClient()
                 .target(targetUrl)
                 .request(MediaType.APPLICATION_JSON_TYPE)
+                .header(AUTHN_TEST_HEADER, SUPER_USER)
```

Comment:
we need this now to pass superuser info for authn.
without this tests will fail due to authentication failure